flight_phase_files/HALO_RF10_20200207_info.yaml

```diff
@@ -22,59 +23,330 @@ remarks:
+  end: 2020-02-07 16:15:23
+  segment_id: HALO-0207_sl1
+  irregularities: []
+  dropsondes:
+    GOOD:
+    - HALO-0207_s37
+    BAD: []
+    UGLY: []
+- kinds: [radar_calibration_wiggle]
+  name: radar wiggle pattern 1
+  start: 2020-02-07 16:15:24
+  end: 2020-02-07 16:16:36
+  segment_id: HALO-0207_rcw1
+  irregularities: []
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds: [radar_calibration_tilted]
+  name: tilted radar calibration maneuver
+  start: 2020-02-07 16:17:25
   end: 2020-02-07 16:26:57
-- kind: straight_leg
-  name: leg2_inbound_NTAS
-  start: 2020-02-07 16:31:11
-  end: 2020-02-07 17:12:38
-- kind: circle
-  name: c4
-  start: 2020-02-07 17:14:54
-  end: 2020-02-07 18:11:04
-- kind: circle_break
-  name: cb4to5
-  start: 2020-02-07 18:11:04
-  end: 2020-02-07 18:29:20
-- kind: circle
-  name: c5
-  start: 2020-02-07 18:29:20
-  end: 2020-02-07 19:22:49
-- kind: circle_break
-  name: cb5to6
-  start: 2020-02-07 19:22:49
-  end: 2020-02-07 19:48:06
-- kind: circle
-  name: c6
-  start: 2020-02-07 19:48:06
-  end: 2020-02-07 20:36:43
-- kind: lidar_calibration
-  name: lidar_cal
+  segment_id: HALO-0207_rct1
+  irregularities: []
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds: [radar_calibration_wiggle]
+  name: radar wiggle pattern 2
+  start: 2020-02-07 16:27:27
+  end: 2020-02-07 16:28:31
+  segment_id: HALO-0207_rcw2
+  irregularities: []
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds: [straight_leg]
+  name: inbound leg from NTAS
+  start: 2020-02-07 16:30:56
+  end: 2020-02-07 17:12:58
+  segment_id: HALO-0207_sl2
+  irregularities: []
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds: [circle]
+  name: circle 4
+  start: 2020-02-07 17:14:50
+  end: 2020-02-07 18:15:53
+  segment_id: HALO-0207_c4
+  irregularities: [traffic_diversion, begin is at first launch due to roll angle at start]
+  dropsondes:
+    GOOD:
+    - HALO-0207_s38
+    - HALO-0207_s39
+    - HALO-0207_s41
+    - HALO-0207_s43
+    - HALO-0207_s44
+    - HALO-0207_s45
+    - HALO-0207_s46
+    - HALO-0207_s47
+    - HALO-0207_s48
+    - HALO-0207_s49
+    BAD:
+    - HALO-0207_s42
+    UGLY:
+    - HALO-0207_s40
+- kinds: [circle_break]
+  name: circle break 4/5
+  start: 2020-02-07 18:15:53
+  end: 2020-02-07 18:28:14
+  segment_id: HALO-0207_cb3
+  irregularities: []
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds: [circle]
+  name: circle 5
+  start: 2020-02-07 18:28:14
+  end: 2020-02-07 19:26:59
+  segment_id: HALO-0207_c5
+  irregularities: []
+  dropsondes:
+    GOOD:
+    - HALO-0207_s50
+    - HALO-0207_s51
+    - HALO-0207_s52
+    - HALO-0207_s53
+    - HALO-0207_s54
+    - HALO-0207_s55
+    - HALO-0207_s56
+    - HALO-0207_s57
+    - HALO-0207_s58
+    - HALO-0207_s59
+    - HALO-0207_s61
+    - HALO-0207_s62
+    BAD: []
+    UGLY: []
+- kinds: [circle_break]
+  name: circle break 5/6
+  start: 2020-02-07 19:26:59
+  end: 2020-02-07 19:39:43
```

Comment:
Ends 2 seconds earlier than start of circle 6.

Reply:
changed it